crates/sui-types/src/base_types.rs

```diff
@@ -537,7 +537,7 @@ impl ObjectDigest {
     }
 
     pub fn encode(&self) -> String {
-        Base64::encode(self.0)
+        Base58::encode(self.0)
```

Comment:
This is quite a risky change. Can we confirm with @joyqvq and other folks whether we should do this? Also feel free to revert this change so that we can merge this quickly

Reply:
I added this function and not sure it is used elsewhere, I will revert this change and add an additional 
base58_encode then.

Reply:
you don't need to worry which encoding to use if you use `serde_json::to_string()`